### PR TITLE
Fix no result view sercher pagination

### PR DESCRIPTION
**Pull request by anetaszynal**
I fix 
1. searching :
- After returning from the results, popular movies/people show up again.
- when we returnig from page 10 of results, we go back to first page of popular movies/people
2. No result view - title is properly located 
3. Pagination - after hover, cursor is replaced by pointer and I set brightness

### CODE REVIEW

File: src/common/Footer/styled.js [hunk 1]
```diff
@@ -22,9 +22,18 @@ export const ChangePageButton = styled.button`
...
+     filter: brightness(105%);
+     cursor: pointer;
+    }
 
     &:disabled{
       background-color: ${({ theme }) => theme.color.grey};
+        &:hover{
+          filter: brightness(100%);
+          cursor: default;
+         }
```
Comment (damdlani) [inline src/common/Footer/styled.js:36]: Great that you remembered to return to default cursor on disabled ;)

File: src/features/movies/MoviesPopular/getFirstMoviePage.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+export const getFirstMoviePage = async (page=1, query) => {
+  if (!query || query.trim() === "") {
+    const response = await fetch(`${URL_ADDRESS}/movie/popular?api_key=${API_KEY}&language=en-US&page=${page}`);
+
+    if (!response.ok) {
+      throw new Error(response.statusText)
+    }
+    return response.json();
+  }
+};
```
Comment (damdlani) [inline src/features/movies/MoviesPopular/getFirstMoviePage.js:12]: Couldn't you just write another condition in already existing function below? It takes the same props. 

`export const getSearchedMovies = async (query, page=1) => {
    if (query || query.trim() !== "") {
        const response = await fetch(`${URL_ADDRESS}/search/movie?api_key=${API_KEY}&language=en-US&page=${page}&query=${query}`);

        if (!response.ok) {
            throw new Error(response.statusText)
        }
        return response.json();
    }
};`

File: src/features/people/PeoplePopular/getFirstPeoplePage.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+export const getFirstPeoplePage = async (page=1, query) => {
+    if (!query || query.trim() === "") {
+        const response = await fetch(`${URL_ADDRESS}/person/popular?api_key=${API_KEY}&language=en-US&page=${page}`);
+
+        if (!response.ok) {
+            throw new Error(response.statusText)
+        }
+        return response.json();
+    }
+};
```
Comment (damdlani) [inline src/features/people/PeoplePopular/getFirstPeoplePage.js:12]: Same as in previous comment

**Review by damdlani — APPROVED** [review]
Great! Check if function can be extended instead of creating new ones

File: src/features/movies/MoviesPopular/getFirstMoviePage.js (hunk 2 above)
Comment (anetaszynal) [reply src/features/movies/MoviesPopular/getFirstMoviePage.js:12]: ok, i'll think about it

File: src/features/people/PeoplePopular/getFirstPeoplePage.js (hunk 3 above)
Comment (anetaszynal) [reply src/features/people/PeoplePopular/getFirstPeoplePage.js:12]: I'll think about it too ;)

**Pull request merged**